examples/js/SkyShader.js

```diff
@@ -73,7 +74,7 @@ THREE.ShaderLib[ 'sky' ] = {
 		"vec3 totalMie(vec3 lambda, float T)",
```

Comment:
if `lambda` is no longer used in this function, it should be removed as a parameter.